exonum/src/api/backends/actix.rs

```diff
@@ -143,6 +143,20 @@ impl ResponseError for api::Error {
     }
 }
 
+impl ResponseError for api::ApiError {
+    fn error_response(&self) -> HttpResponse {
+        let mut response = match self.http_code {
+            HttpCode::BadRequest => HttpResponse::BadRequest(),
+            HttpCode::NotFound => HttpResponse::NotFound(),
+            HttpCode::NotImplemented => HttpResponse::NotImplemented(),
+        };
+
+        response
+            .header(header::CONTENT_TYPE, "application/problem+json")
+            .body(self.to_string())
```

Comment:
I think it's inappropriate use of `to_string`; it could make more sense to use `serde_json::to_string` directly.